examples/plot_wavelet.py

```diff
@@ -67,3 +67,50 @@
 axs[1, 1].set_title("DWT2 coefficients (zeroed)")
 axs[1, 1].axis("tight")
 plt.tight_layout()
+
+###############################################################################
+# Let us now try the same with a 3D volumetric model, where we use the
+# N-dimensional DWT. Again, we only retain a quarter of the coefficients of
+# the DWT.
+
+nx = 128
+ny = 256
+nz = 128
+
+x = np.arange(nx)
+y = np.arange(ny)
+z = np.arange(nz)
+
+xx, yy, zz = np.meshgrid(x, y, z, indexing="ij")
+# Generate a 3D model with two block anomalies
+m = np.ones_like(xx, dtype=float)
+block1 = (xx > 10) & (xx < 60) & (yy > 100) & (yy < 150) & (zz > 20) & (zz < 70)
+block2 = (xx > 70) & (xx < 80) & (yy > 100) & (yy < 200) & (zz > 10) & (zz < 50)
+m[block1] = 1.2
+m[block2] = 0.8
+Wop = pylops.signalprocessing.DWTND((nx, ny, nz), wavelet="haar", level=3)
+y = Wop * m
+
+yf = y.copy()
+yf.flat[y.size // 4 :] = 0
+iminv = Wop.H * yf
+
+ratio = 0.1
```

Comment:
Looks like this is overwriting the lines 94-96 and doing the same for a ratio of 10%. I suggest to keep either of the two and if keeping this one to modify the text above from

```
Again, we only retain a quarter of the coefficients of the DWT.
```

to something like:

```
This time we only retain 10% of the coefficients of the DWT.
```

Reply:
You are absolutely right! I copied over the 2D and forgot about deleting those lines. Sorry about that!